### PR TITLE
Improve param schema retrieval

### PR DESCRIPTION
**Pull request by rvegas**

### CODE REVIEW

File: sdk/action-schema.js [hunk 1]
```diff
@@ -391,7 +393,13 @@ class ActionSchema extends Schema {
       throw new Error(`Cannot resolve schema for parameter: ${name}`);
     }
 
-    return this._params[name];
+    let schema = null;
+    this._params.forEach((param) => {
+      if (param._name === name) {
```
Comment (nullproxy) [inline sdk/action-schema.js]: If this matches shouldn't this break out of the loop?

Comment (rvegas) [reply sdk/action-schema.js]: indeed, done

**Comment by rvegas** [comment]
I've added more tests.

**Comment by rvegas** [comment]
I'm merging since I've got no reviews

**Pull request merged**